mmdeploy/backend/onnxruntime/wrapper.py

```diff
@@ -50,9 +50,17 @@ def __init__(self,
             logger.warning('The library of onnxruntime custom ops does '
                            f'not exist: {ort_custom_op_path}')
         device_id = parse_device_id(device)
-        providers = ['CPUExecutionProvider'] \
-            if device == 'cpu' else \
-            [('CUDAExecutionProvider', {'device_id': device_id})]
+        providers = []
+        if device == 'cpu':
+            providers.append('CPUExecutionProvider')
+        else:
+            providers.append(('CUDAExecutionProvider', {
```

Comment:
As far as I know, cpu provider can co-exist with cuda provider, ONNXRuntime will fall back to cpu if the cuda implementation of the op is not provided.
Cpu provider can be placed in the providers list even when we use gpu device. (I am not sure if the order of the provider matters)

Reply:
According to the official documentation, the order of providers is the same as preference. Therefore, when the device is cuda, tensorrt and cuda executor are included, and the cpu executor is added at the end as the default value. (https://onnxruntime.ai/docs/execution-providers/)